src/core/exception.d

```diff
@@ -315,14 +315,17 @@ unittest
 }
 
 
+deprecated("UnicodeException is now an Error.")
+alias UnicodeException = UnicodeError;
+
 /**
  * Thrown on a unicode conversion error.
  */
-class UnicodeException : Exception
+class UnicodeError : Error
```

Comment:
Breaking change. How would users fix their code? In fact it may pass all of tests untill it hits bad encoding in the wild and crashes.